nbgitpuller/plugins/zip_puller.py

```diff
@@ -0,0 +1,79 @@
+from .plugin_helper import fetch_files
+from .plugin_helper import handle_files_helper
+import pluggy
+
+hookimpl = pluggy.HookimplMarker("nbgitpuller")
+TEMP_DOWNLOAD_REPO_DIR = ".temp_download_repo"
+CACHED_ORIGIN_NON_GIT_REPO = ".origin_non_git_sources"
+
+
+# handles standard web addresses(not google drive or dropbox)
+class ZipSourceWebDownloader(object):
```

Comment:
These should just be functions, and exposed via entrypoints

Reply:
yes made this change -- good deal